core/arch/proc_api.h

```diff
@@ -463,6 +463,25 @@ DR_API
 bool
 proc_has_feature(feature_bit_t feature);
 
+#if defined(AARCH64) && (defined(BUILD_TESTS) || defined(STANDALONE_DECODER))
```

Comment:
I think we have to remove the requirement for the STANDALONE_DECODER define for the use case of using both offilne decoding and managed execution from the same instance of the DR library.  That is our use case, because we have to statically link all libraries into the executable and so cannot do a separate build of DR for the two use cases: thus we cannot define STANDALONE_DECODER.  So I think we have to expose this in all builds -- or possibly add `|| STATIC_LIBRARY`.

Reply:
Deleted.